### PR TITLE
RavenDB sample(s) on Linux

### PR DESCRIPTION
**Pull request by DavidBoike**

### CODE REVIEW

File: samples/ravendb/simple/sample.md [hunk 1]
```diff
@@ -29,20 +29,13 @@ This sample shows a simple Client + Server scenario.
 
 ### Raven Config
 
-Configure the endpoint to use RavenDB persistence.
+Configure the endpoint to use RavenDB persistence. The Url/Port may need to be changed to match the database in use.
```
Comment (bording) [inline samples/ravendb/simple/sample.md]: Why are "Url" and "Port" capitalized?

**Comment by bording** [comment]
Oh, one other thing. The new v5 Core solution needs a R# settings file.

File: samples/ravendb/simple/sample.md (hunk 1 above)
Comment (DavidBoike) [reply samples/ravendb/simple/sample.md]: Url will be capitalized in the snippet, though that technically contains the port too. Perhaps just "The Url may be..." would be better.

Comment (bording) [reply samples/ravendb/simple/sample.md]: 👍 For "The Url" since that includes the port.

**Pull request merged**